### PR TITLE
Change post #116 to potentially change serialization of `java.nio.file.Path`

### PR DESCRIPTION
**Pull request by cowtowncoder**

### CODE REVIEW

**Comment by cowtowncoder** [comment]
@jhonnen So, this would change behavior to, I think, what you are asking. WDYT?
One more possibility would be addition of a new `JSON.Feature` to control behavior (choose one of 2, I suspect).

Either way, I am hoping to finalize handling soon, for upcoming 2.17.0-rc1 release.

File: jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/SimpleValueReader.java [hunk 1]
```diff
@@ -278,7 +278,8 @@ protected Path _readPath(JsonParser p) throws IOException {
         }
         String v = p.getValueAsString();
         try {
-            return Paths.get(new URI(v));
+//            return Paths.get(new URI(v));
+            return Paths.get(new File(v).toURI());
```
Comment (jhonnen) [inline jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/SimpleValueReader.java:282]: why not simplify to
```suggestion
            return Paths.get(v);
```

File: jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/JSONWriter.java [hunk 2]
```diff
@@ -789,7 +789,8 @@ protected String dateToString(Date v) {
      * @since 2.17
      */
     protected String pathToString(Path value) {
-        return value.toUri().toString();
+//        return value.toUri().toString();
+        return value.toString();
```
Comment (jhonnen) [inline jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/JSONWriter.java:793]:
```suggestion
        return value.toString();
```

**Review by jhonnen — APPROVED** [review]
that's the behavior I expected 👍

**Pull request merged**